examples/blueprint-construct/index.ts

```diff
@@ -127,9 +127,6 @@ export default class BlueprintConstruct {
                     value: "test",
                     effect: "NoSchedule",
                 }],
-                amiSelector: {
-                    "karpenter.sh/discovery/MyClusterName": '*',
```

Comment:
Is there a reason why this is removed. Dont we dont need an `amiSelector?

Reply:
`amiSelector` is used in case of custom AMIs. In this case if we run this through CI (or my own account which has none), Karpenter will deploy but the Node template will error out when calling upon the fleet.

Reply:
Sounds good!